### ISSUE TITLE
'Use as local storage' shows no files

### ISSUE THREAD

**Issue by resetter** [wa2c/cifs-documents-provider#58]
The  'use as local storage' function in the app does not show files in any apps.
Using document provider function the files show up as expected, also I have made sure to enable the option and have rebooted.
Any suggestions?

**Comment by RebelliousX** [comment]
@resetter It is working fine here. What are you trying to do? What are the steps to reproduce the problem? Did you forget to fill the Bug Report template when posting this issue?

Use as local storage is only meant for apps asking for files to be on the internal storage only. It is just a way to fool these apps into thinking it is dealing with an internal storage. Enabling this will have ZERO effects on anything else. Your issue is something else.

When you created the CIFS connection, did you tap on "Check Connectivity" to make sure you did not make any typo with the username or password?

**Comment by resetter** [comment]
I am using version 2.0.0 and the device is the AYN Odin2 (Max)
I want to access an SMB share on my Mac mini, the share is fine and accessible via the document provider using Mixplorer on android. However, when I try to access the share as a 'virtual sd card' I can see a folder inside with the correct name of my computer as defined in cifs app, but when I enter the folder there are no files, empty directory.

**Comment by resetter** [comment]
Hey again, just to bump this thread, the latest release still doesnt seem to mitigate the problem, are you guys able to 'see' the virtual sd card?

**Comment by RebelliousX** [comment]
@resetter Just curious about your Mac settings, I just, are you sure you ticked the check box next to the account you are using for SMB share? like in this photo (not mine), I can't use one of my Macs now to test. Also if you are using VPN software try disabling that momentarily and test again.
![Photo](https://github.com/wa2c/cifs-documents-provider/assets/1888186/1626dbc9-dadd-472c-b8ee-efbe86f8ec5a)

**Comment by RebelliousX** [comment]
@resetter Please double check your Share settings in you Mac Mini. Are you sure you ticked the check box next to the user account you are using for the share? If you are using a VPN service or software, try disabling that momentarily and test again. See this photo (not mine), I can't use one of my Macs right now to test.
![Photo](https://github.com/wa2c/cifs-documents-provider/assets/1888186/cc99b3a7-f7e7-4eff-ab64-c0be7be57000)

**Comment by resetter** [comment]
I can confirm that the share is setup correctly and is accessible via SMB from my android device. However, the 'virtual' sd card option shows no sign, I have checked the root folder in android, such as in the storage folder, all I can see is my physical sd card and internal storage. Am I missing something?

**Comment by c0ffincolors** [comment]
I don't think you need root access, I am also trying to set it up in Winlator. The issue is that Winlator does not show the path once it's created, it should theoretically be /storage/emulated/0/Android/data/com.wa2c.android.cifsdocumentprovider/yourshare
But Winlator even though it opens File Picker, does not add the path to the program, and so I have no way of understanding how the path actually is displayed to the app. I modded Winlator for All Files Access support, which helps with USB storage, but I also would like to understand how to identify the paths
